### ISSUE TITLE
Churn Model

### ISSUE THREAD

**Issue by amrutadpatil2001-droid** [amrutadpatil2001-droid/Python_Projects#16]
from sklearn.ensemble import RandomForestClassifier
from sklearn.model_selection import train_test_split
from sklearn.metrics import accuracy_score, f1_score

def train_churn_model(df, target='Churn'):
    
    X = df.drop(columns=[target])
    y = df[target]
    
    X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2)
    
    model = RandomForestClassifier()
    model.fit(X_train, y_train)
    
    preds = model.predict(X_test)
    
    return {
        "model": model,
        "accuracy": accuracy_score(y_test, preds),
        "f1": f1_score(y_test, preds)
    }
